### PR TITLE
Fix debian package

### PR DESCRIPTION
**Pull request by Trophime**
**Description of the Pull Request (PR):**

Update debian files for 2.3 release
The test still cannot be performed when building the package

**This fixes or addresses the following GitHub issues:**

- Ref: #699

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [x] I have tested this PR locally with a `make test`
- [x] I have added myself as a contributor to the [Author's file](https://github.com/singularityware/singularity/blob/master/AUTHORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -2,16 +2,20 @@ Source: singularity
 Section: admin
 Priority: optional
 Maintainer: Dave Love <fx@gnu.org>
-Build-Depends: debhelper (>= 9), dh-autoreconf
+Build-Depends: debhelper (>= 9), dh-autoreconf,
+ python-dev (>= 2.6.6-3~),
```
Comment (yarikoptic) [inline debian/control:6]: FWIW AFAIK no python-dev is needed, ie enough of python

Comment (gmkurtzer) [reply debian/control:6]: Yep, as @yarikoptic mentioned.

**Comment by gmkurtzer** [comment]
Can this PR be resubmitted to the development branch please?

**Comment by yarikoptic** [comment]
FWIW -- current state of the "official" Debian packaging could be found in this repository https://anonscm.debian.org/git/collab-maint/singularity-container.git

Cheers,

**Comment by gmkurtzer** [comment]
@Trophime this looks great, can you resubmit this to the development branch?

**Pull request closed without merging**